config/routes.rb

```diff
@@ -42,5 +42,6 @@
       put "toggle_admin", on: :member
     end
   end
+  match "(errors)/:status", to: "errors#show", constraints: { status: /\d{3}/ }, via: :all
```

Comment:
no i can't. This match is for routing the (errors) to the errors controller. By (errors) I mean 500, 400 errors, like when an exception is raised.